src/main/java/com/example/App.java

```diff
@@ -1,10 +1,33 @@
 package com.example;
 
-/**
- * Hello world!
- */
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.servlet.ServletContextHandler;
+import org.eclipse.jetty.servlet.ServletHolder;
+
 public class App {
-    public static void main(String[] args) {
-        System.out.println("Hello World!");
+
+    public static void main(String[] args)
+            throws Exception {
+        ServletContextHandler context = new ServletContextHandler(ServletContextHandler.SESSIONS);
+
+        Server jettyServer = new Server(8080);
+        jettyServer.setHandler(context);
+
+        ServletHolder jerseyServlet = context.addServlet(
+                org.glassfish.jersey.servlet.ServletContainer.class, "/api/*");
+        jerseyServlet.setInitOrder(0);
+
+        // Tells the Jersey Servlet which REST service/class to load.
+        jerseyServlet.setInitParameter(
+                "jersey.config.server.provider.packages",
+                "com.example");
```

Comment:
I would move the authentication endpoint and filter into rest package, as they are part of the api.